fullstack-cert/html-projects/lab-multimedia-player/user-stories.md

```diff
@@ -0,0 +1,11 @@
+1. You should have an `<h1>` element for the main title of the page.
+2. You should have three `section` elements.
+3. Inside the first `section` element, you should have an `<h2>` element for the title of song playing.
+4. Below the `<h2>` element, you should have an `audio` element with `controls` attribute and an `aria-label` attribute.
+5. Inside the `audio` element, you should have a `source` element with a `src` attribute pointing to an audio file and a `type` attribute.
+6. Inside the second `section` element, you should have an `<h2>` element for the title of the video playing.
+7. Below the `<h2>` element, you should have a `video` element with `controls`, `width` attributes and an `aria-label` attribute.
+8. Inside the `video` element, you should have a `source` element with a `src` attribute pointing to a video file and a `type` attribute.
+9. Below the `source` element, you should have a `track` element with a `src` attribute pointing to a subtitles file and a `kind` attribute, a `srclang` attribute and a `label` attribute.
+10. Inside the third `section` element, you should have an `<h2>` element for the title of the section eg. "Transcript".
+11. Below the `<h2>` element, you should have a `p` element with the transcript of the video.
```

Comment:
When mentioning element names, we don't include `<>`
So all instances of those should be removed in the user stories here